app/views/provider/admin/dashboards/widgets/_service_top_traffic.html.slim

```diff
@@ -3,12 +3,16 @@ article.DashboardWidget id = widget.id
   h1.DashboardWidget-title Top Applications
   span.DashboardWidget-intro Apps with consistently high traffic in the last 30 days
 
+  - widget_service_id = widget.params['service_id']
+
   - if widget.loaded?
     - if widget.items.present?
       ol.DashboardWidgetList
         - widget.items.each do |item|
           li class="DashboardWidgetList-item DashboardWidgetList-item--#{item.state}"
-            = link_to (application = item.application) ? admin_service_application_path(application.service, application) : '#', class: 'DashboardWidgetList-link' do
+            - show_application_path_exists = ((application = item.application) && widget_service_id)
```

Comment:
So this is not a `Cinstance` or any `Contract`
It comes from Top traffic Query which are data from redis.